src/us/kbase/workspace/database/ObjectInformation.java

```diff
@@ -149,12 +149,22 @@ public Optional<UncheckedUserMetadata> getUserMetaData() {
 		return Optional.ofNullable(usermeta);
 	}
 	
-	/** Returns the user supplied and automatically generated metadata for the object as a map
-	 * or null if no user metadata was provided.
+	/** Returns the user supplied and automatically generated metadata for the object as a map.
+	 * 
+	 * @param nullIfEmpty return null rather than an empty map if there is no metadata available.
```

Comment:
does the `@return` description below need to be updated?

Reply:
Fixed